Makefile

```diff
@@ -1,10 +1,71 @@
+.PHONY: all build clean test
+all: build
+
+# HELP sourced from https://gist.github.com/prwhite/8168133
+
+# Add help text after each target name starting with '\#\#'
+# A category can be added with @category
+
+HELP_FUNC = \
+    %help; \
+    while(<>) { \
+        if(/^([a-z0-9_-]+):.*\#\#(?:@(\w+))?\s(.*)$$/) { \
+            push(@{$$help{$$2}}, [$$1, $$3]); \
+        } \
+    }; \
+    print "usage: make [target]\n\n"; \
+    for ( sort keys %help ) { \
+        print "$$_:\n"; \
+        printf("  %-30s %s\n", $$_->[0], $$_->[1]) for @{$$help{$$_}}; \
+        print "\n"; \
+    }
```

Comment:
should we consider moving to https://taskfile.dev/installation/ in the future?
Looks better in terms of readability and the flow

Reply:
Makefile is used across projects and is ubiquitous. We can discuss with a larger group on this.